Dockerfile.agent

```diff
@@ -38,6 +34,7 @@ COPY --from=builder /build/views ./views
 COPY --from=builder /build/public ./public
 COPY --from=builder /build/app ./app
 COPY --from=builder /build/fakepasswd /etc/passwd
+COPY --from=builder /etc/ssl/certs /etc/ssl/certs
```

Comment:
Why does this change require copying certs from the builder image?

Reply:
As the `scratch` image is pretty basic, ssl verification can't be done on it. Therefore we get them from the builder stage and pass to the produced app image. Otherwise we get an error - `x509: certificate signed by unknown authority`.

Of course the other option is to switch `scratch` to `alpine` which works well too.